farmdata2_modules/fd2_tabs/resources/DateRangeSelectionComponent.spec.comp.js

```diff
@@ -94,10 +132,10 @@ describe('date range selection component', () => {
         })
 
         it ('change setEndDate prop to 2021-09-09', () => {
-            expect(comp.vm.latestStartDate).to.equal('2021-01-01')
+            expect(comp.vm.latestStartDate).to.equal('2021-12-31')
             cy.wrap(comp.setProps({ setEndDate: '2021-09-09' }))
```

Comment:
One of my earlier commits for the test did not go into my earlier pull request #486. I included the changes here.